Extensions/quick_tags.js

```diff
@@ -297,9 +302,6 @@ XKit.extensions.quick_tags = new Object({
 		$("#xkit-quick-tags-window").remove();
 		$("body").append(m_html);
 
-		$("#xkit-tag-input").bind("focus", XKit.extensions.quick_tags.cancel_menu_close);
-		$("#xkit-tag-input").bind("blur", XKit.extensions.quick_tags.menu_close);
```

Comment:
That's the goal. The behaviour is now similar to one click postage, i.e. if one click outside of the popup, it closes, *unless* the focus was given to the input area, in which case it won't close (in order to do some copy-paste for example)